aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,302 @@
+                Models.Cat cat = new Models.Cat();               // Simplified NS
+                cat.name = "spot";
+                System.Console.WriteLine(cat.name);
+            }
+        }
+    }
+
+When you create resource files based on a type, the resource file path must include the type with the simplified namespace. In the sample project, the default namespace is ``Localization.StarterWeb``. The ``HomeController`` reference with a simplified namespace is ``Controllers.HomeController``, therefore the French resource file must be named *Controllers.HomeController.fr.resx*. The ``ConfigureServices`` method sets the ``ResourcesPath`` to "Resources", so the project relative path for the home controllers French resource file is *Resources/Controllers.HomeController.fr.resx*. Alternatively, you can use folders to organize resource files. For the home controller, the path would be *Resources/Controllers/HomeController.fr.resx*. If you don't use the ``ResourcesPath`` option, the *.resx* file would go in the  project base directory. The resource file for ``HomeController`` would be named *Controllers.HomeController.fr.resx*.
+
+Resource files using ``@inject IViewLocalizer`` in Razor views follow a similar pattern. The resource file for a view can be named using either dot naming or file naming. Razor view resource files mimic the path of their associated view file. Assuming we set the ``ResourcesPath`` to "Resources", the French resource file associated with the *Views/Book/About.cshtml* view could be either of the following:
+
+- Resources/Views/Home/About.fr.resx
+- Resources/Views.Home.About.fr.resx
+
+If you don't use the ``ResourcesPath`` option, the *.resx* file for a view would be located in the same folder as the view.
+
+Generating resource files with Visual Studio
+-----------------------------------------------
+
+If you create a resource file in Visual Studio without a culture in the file name (for example, *MyStrings.resx*), Visual Studio will create a C# class with a property for each string.  That’s usually not what you want with ASP.NET Core; you typically don’t have a default resx resource file (A resx file without the culture name). We suggest you create the .resx file with a culture name (for example *MyStrings.fr.resx*). When you create a .resx file with a culture name, Visual Studio will not generate the class file.
+
+If you remove the ".fr" culture designator AND you have the culture set to French (via cookie or other mechanism), the default resource file is read and strings are localized. This is expected behavior. \*.resx designates a default or fallback resource, so that when nothing meets your requested culture you’re served \*.resx. See `ASP.NET Web Page Resources Overview <https://msdn.microsoft.com/en-us/library/ms227427.aspx>`__ for more information. If you want to just return the key when missing a resource for the requested culture you must not have a default resource file.
```

Comment:
What's the relevant content in https://msdn.microsoft.com/en-us/library/ms227427.aspx ? That's all Web Forms localization, which has close to zero overlap with the content here, no?